### PR TITLE
Surviving Burnout

### PR DESCRIPTION
**Pull request by squinones**
An article that's been rattling around in my head for a bit. I'd love your thoughts. :)

### CODE REVIEW

**Comment by erikaheidi** [comment]
Absolutely amazing, I'm just saving it for the new website release! Thank you @squinones for this wonderful article, it will help a lot of people. The release is gonna happen tomorrow or the day after, I'll ping you!

File: source/_posts/2015-10-07-surviving-burnout.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+When I found myself daydreaming about crashing through a guardrail on my way to work one morning, I realized something had to change. This job, I thought, was killing me. I had to save myself, so I quit, and I spent the next few years draining my savings account. It was an incredibly destructive period in my life, but I did manage to do one very positive thing: I saw a therapist. It was through therapy that I started to understand what had happened to me and how I had gone from being productive and happy to being so miserable I could barely force myself out of bed. I learned about burnout.
+
+### Recognizing Burnout
+In the simplest terms, burnout is exhaustion. It can be physical or mental, but it's almost always the product long-term exposure to heavy stress, or situations that are physically or emotionally taxing. Burnout is where we get to when we've pushed ourselves to the limit and then kept pushing. It's when a two-week crunch turns in to six months of twelve-hour days and six-day weeks. It's what happens when we stop resting behaving like we're in a life-or-death struggle for survival instead of just working a job.
```
Comment (meadsteve) [inline source/_posts/2015-10-07-surviving-burnout.md]: I think there's an ```of``` missing from in between  ```product long-term```

**Comment by squinones** [comment]
Thanks, good catch, @meadsteve!

**Comment by meadsteve** [comment]
No problem!  It's a great article. Really glad you've shared it.

**Pull request merged**